core/src/test/java/com/graphhopper/routing/ch/CHTurnCostTest.java

```diff
@@ -747,64 +747,6 @@ public void testFindPath_loopEdge() {
         compareCHQueryWithDijkstra(pch, 0, 4);
     }
 
-    @Test
```

Comment:
Can you move this back below issue1593_simple ? looks like nothing really changed here ?

Reply:
Ah, sure. Was just an ugly merge conflict where I had many duplicated methods and I obviously picked the wrong that I removed ;)